src/core/iomux/epfd_info.cpp

```diff
@@ -89,10 +87,6 @@ epfd_info::epfd_info(int epfd, int size)
 
     xlio_stats_instance_create_epoll_block(m_epfd, &(m_stats->stats));
 
-    // Register this socket to read nonoffloaded data
-    g_p_event_handler_manager->update_epfd(m_epfd, EPOLL_CTL_ADD,
```

Comment:
[question] Is "issue: 3502635 Avoid epoll OS polling through internal thread" generic enough to port to vNext?

Reply:
Yes. Rebased and ported from vNext.